html-forms/index.html

```diff
@@ -15,36 +15,34 @@ <h1>Coding Form!</h1>
       <div>
         <label for="coding-select">Why is coding cool?</label>
         <div>
-          <select id="why-is-coding-cool-select">
+          <select id="coding-select">
             <option value="">Select an option</option>
             <option value="interesting">Because it's interesting</option>
             <option value="fun">Because it's fun</option>
             <option value="powerful">Because it's powerful</option>
           </select>
         </div>
         <div>
-          <label for="favorite-language">Favorite Language (choose one)</label>
+          <p>Favorite Language (choose one)</p>
           <div>
             <input type="radio" id="HTML" name="language" value="HTML" />
             <label for="HTML">HTML</label>
             <input type="radio" id="CSS" name="language" value="CSS" />
             <label for="CSS">CSS</label>
             <input
               type="radio"
-              id="JavaScript"
+              id="javascript"
               name="language"
               value="JavaScript" />
             <label for="JavaScript">JavaScript</label>
           </div>
           <div>
-            <label for="sign-up"
-              >Would you like to sign up for more information?</label
-            >
+            <p>Would you like to sign up for more information?</p>
             <div></div>
             <input type="checkbox" id="sign-me-up" name="sign me up" />
             <label for="sign-me-up">Sign me up!</label>
             <div></div>
-            <label for="comments">Comments</label>
+            <label for="comments" id="comments">Comments</label>
```

Comment:
Wrong place. The `id="comments"` needs to go on the `textarea`. The `for` attribute of the `label` matches the `id` of an `input`, `textarea`, etc.